src/configs/git_status.rs

```diff
@@ -18,6 +18,7 @@ pub struct GitStatusConfig<'a> {
     pub modified: &'a str,
     pub staged: &'a str,
     pub untracked: &'a str,
+    pub ignore_submodules: bool,
```

Comment:
Since this PR introduces a new `ignore_submodules` option, we would need to have documentation and tests added for this to merge.